tests/ord-service/tests/subscription_flow_test.go

```diff
@@ -42,6 +42,58 @@ import (
 	"github.com/tidwall/gjson"
 )
 
+func TestSelfRegisterFlow(stdT *testing.T) {
+	t := testingx.NewT(stdT)
+	t.Run("TestSelfRegisterFlow flow: label definitions of the parent tenant are not overwritten", func(t *testing.T) {
+		ctx := context.Background()
+
+		// defaultTenantId is the parent of the subaccountID
```

Comment:
Currently the `tenant.TestTenants.GetDefaultTenantID()` is not parent of the `tenant.TestProviderSubaccount`. We can set it in the values.yaml file